test/controllers/api/accounts_controller_test.rb

```diff
@@ -273,6 +273,74 @@ class Api::AccountsControllerTest < ActionController::TestCase
       assert_equal 'test', account['cleartext_username']
       assert_equal 'password', account['cleartext_password']
     end
+
+  end
+
+  context '#update' do
+    test 'update account with valid params structure' do
+      request.headers['Authorization-User'] = bob.username
+      request.headers['Authorization-Password'] = Base64.encode64('password')
+
+      account = accounts(:account1)
+
+      request_account = {
```

Comment:
rename to account_params

make sure to update the username, password ! that's what really matters here ! it's the most important feature of cryptopus, isn't it ? storing and retrieving credentials :man_dancing: